nni/retiarii/oneshot/pytorch/strategy.py

```diff
@@ -0,0 +1,126 @@
+"""Strategy integration of one-shot.
```

Comment:
copyright

Reply:
Reminds me, I'll add a pipeline rule for copyright at some point.